README.rst

```diff
@@ -1,26 +1,26 @@
 About Plone
 ===========
 
-Plone is a mature, secure and user-friendly Content Management System (CMS).
+Plone is a mature, secure, and user-friendly Content Management System (CMS).
 
-Plone - and the Open Source community behind it - aggregates more than 15 years experience in content management.
+Plone - and the Open Source community behind it - aggregates more than 15 years of experience in content management.
 It offers all major features expected by a modern CMS out-of-the-box.
 
-Lots of customizations can be made trough-the-web, such as creating content types, themes, workflows and much more.
+Lots of customizations can be made trough-the-web, such as creating content types, themes, workflows, and much more.
 Pushed one step further Plone can be used as a framework on which to build custom CMS-like solutions.
 
 Plone works as a
 
 - full-featured classical server-side rendered CMS,
-- headless CMS offering all features as a REST API.
+- headless CMS offers all features as a REST API.
```

Comment:
The change on this line is wrong. Now it no longer forms a sentence together with the line before the list.

The rest of the PR is good.

Reply:
Can you explain... as I was unable to get your point.... @mauritsvanrees

Reply:
Plone works as a

- full-featured classical server-side rendered CMS,
- headless CMS that offers all features as a REST API.

Reply:
@rohnsha0 With your change, it reads like this:

- Plone works as a full-featured classical server-side rendered CMS,
- Plone works as a CMS offers all features as a REST API.

The second sentence is wrong. The original text works. But since you changed it, this may mean it is not clear. Is the way @mtrebron it says working for you, inserting the word `that`?

Alternatively, we could also rewrite this into a proper paragraph instead of two bullet points. Maybe something like this:

"Plone works as a full-featured classical server-side rendered CMS, and as a headless CMS that offers all features as a REST API."

Reply:
@mauritsvanrees Actually in my opinion, keeping the bullets is more better as the same bulleting format is followed in the later part of the file!
So I do think, changes suggested by @mtrebron (adding that) would be favourable as it matches and would be better to read!

However, I would like to here from you too @mauritsvanrees before doing anything!

Reply:
That would be fine. Can you make a PR? Or actually, if you add this one word on master and push it directly, that would be fine too.

Reply:
Naaah... I'm thinking of a PR!

Reply:
@mauritsvanrees done! #24